core/src/models/Export.ts

```diff
@@ -191,12 +197,15 @@ export class Export extends Model {
     if (error["errorLevel"]) this.errorLevel = error["errorLevel"];
 
     this.retryCount++;
+
     if (this.retryCount >= maxExportAttempts) {
       this.state = "failed";
       this.sendAt = null;
     } else if (this.errorLevel === "info") {
       this.state = "failed";
     } else {
+      this.state = "pending";
+      this.exportProcessorId = null;
```

Comment:
You may want to test this explicitly - if there was ExportProcessor fo this Export, and the export Failed (somehow?), the exportProcessorId should be removed